src/wp-admin/includes/file.php

```diff
@@ -1946,6 +1946,75 @@ function copy_dir( $from, $to, $skip_list = array() ) {
 	return true;
 }
 
+/**
+ * Moves a directory from one location to another.
+ *
+ * Recursively invalidates OPcache on success.
+ *
+ * If the renaming failed, falls back to copy_dir().
+ *
+ * Assumes that WP_Filesystem() has already been called and setup.
+ *
+ * @since 6.2.0
+ *
+ * @global WP_Filesystem_Base $wp_filesystem WordPress filesystem subclass.
+ *
+ * @param string $from      Source directory.
+ * @param string $to        Destination directory.
+ * @param bool   $overwrite Overwrite destination.
+ *                          Default is false.
+ * @return true|WP_Error True on success, WP_Error on failure.
+ */
+function move_dir( $from, $to, $overwrite = false ) {
+	global $wp_filesystem;
+
+	if ( ! $overwrite && $wp_filesystem->exists( $to ) ) {
+		return new WP_Error(
+			'to_directory_already_exists_move_dir',
+			sprintf(
+				/* translators: %s: The '$to' argument name. */
+				__( '%s already exists.' ),
```

Comment:
A similar string already exists but it doesn't include the destination.

https://github.com/WordPress/wordpress-develop/blob/5836a438084356ebd232ffc152e18a6cf1c4f16f/src/wp-admin/includes/class-wp-upgrader.php#L163

If the folder name is included, I'd make it a little clearer with something along the lines of `The destination folder, %s, already exists.`

Reply:
The folder name is not included here, but we can certainly use `__( 'Destination folder already exists.' )` to get the benefit of an already translated string. Does that sound good to you?

Reply:
Updated and force-pushed. 👍